### PR TITLE
docs: upadate Chinese installation docs

### PR DESCRIPTION
**Pull request by hf400159**
### Description

Updated Chinese installation documentation.
This PR is only responsible for the internationalization and synchronization of English changes from https://github.com/apache/apisix/pull/6964.

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/how-to-build.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ---
-title: 如何构建 Apache APISIX
+title: APISIX 安装教程
```
Comment (SylviaBABY) [inline docs/zh/latest/how-to-build.md:2]:
```suggestion
title: APISIX 安装指南
```

File: docs/zh/latest/how-to-build.md [hunk 2]
```diff
@@ -21,345 +26,220 @@ title: 如何构建 Apache APISIX
...
 
-## 步骤 1 安装 Apache APISIX
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-你可以通过 RPM 仓库、Docker、Helm Chart、源码包、源码包（LTS 版本）等多种方式来安装 Apache APISIX。请在以下选项中选择其中一种执行。
+本文将介绍如何在你的环境中安装并运行 APISIX。
 
-### 通过 RPM 仓库安装（CentOS）
+关于如何快速运行 Apache APISIX，请参考 [入门指南](./getting-started.md)。
```
Comment (SylviaBABY) [inline docs/zh/latest/how-to-build.md:34]:
```suggestion
关于如何快速运行 Apache APISIX，请参考[入门指南](./getting-started.md)。
```

File: docs/zh/latest/how-to-build.md [hunk 3]
```diff
@@ -21,345 +26,220 @@ title: 如何构建 Apache APISIX
...
 
-> 这种安装方式适用于 CentOS 7 和 CentOS 8 操作系统。
+## 安装 etcd
 
-如果尚未安装 OpenResty 的官方 RPM 仓库，请使用以下命令自动安装 OpenResty 和 Apache APISIX 的 RPM 仓库。
+APISIX 使用 [etcd](https://github.com/etcd-io/etcd) 作为配置中心，来保存和同步配置。在安装 APISIX 之前，你需要在你的主机上安装 etcd。
```
Comment (SylviaBABY) [inline docs/zh/latest/how-to-build.md:38]:
```suggestion
APISIX 使用 [etcd](https://github.com/etcd-io/etcd) 作为配置中心进行保存和同步配置。在安装 APISIX 之前，需要在你的主机上安装 etcd。
```

File: docs/zh/latest/how-to-build.md [hunk 4]
```diff
@@ -21,345 +26,220 @@ title: 如何构建 Apache APISIX
...
 
-```shell
-sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm
-```
+如果你在安装 APISIX 时选择 Docker 或 Hlem 安装，etcd 将会自动安装。如果你选择其他方法或者需要手动安装 APISIX，请参考以下步骤安装 etcd：
```
Comment (SylviaBABY) [inline docs/zh/latest/how-to-build.md:40]:
```suggestion
如果你在安装 APISIX 时选择了 Docker 或 Helm 安装，则 etcd 将会自动安装；如果你选择其他方法或者需要手动安装 APISIX，请参考以下步骤安装 etcd：
```

File: docs/zh/latest/how-to-build.md [hunk 5]
```diff
@@ -21,345 +26,220 @@ title: 如何构建 Apache APISIX
...
 ```
 
-### 停止运行 Apache APISIX
+:::
 
-优雅停机 `apisix quit` 和强制停机 `apisix stop` 都可以停止运行 Apache APISIX。建议您优先选择优雅停机的方式停止 Apache APISIX，因为这种停止方式能够保证 Apache APISIX 完成了已经接受到的请求之后再停止；而强制停机则是立即停止 Apache APISIX，在这种情况下，Apache APISIX 接收到但未完成的请求会随着强制停机一并停止。
+### 通过 RPM 包离线安装：
 
-执行优雅停机的命令如下所示：
+下载 APISIX 离线 RPM 包到 `apisix` 文件夹：
```
Comment (SylviaBABY) [inline docs/zh/latest/how-to-build.md:173]:
```suggestion
将 APISIX 离线 RPM 包下载到 `apisix` 文件夹：
```

File: docs/zh/latest/how-to-build.md [hunk 6]
```diff
@@ -21,345 +26,220 @@ title: 如何构建 Apache APISIX
...
-如果使用的操作系统为 CentOS，且在步骤 2 中通过 RPM 包安装 Apache APISIX，配置文件已经自动安装到位，你可以直接运行以下命令：
+如果你是通过 RPM 包安装 APISIX，配置文件已经自动安装，你可以直接使用以下命令：
 
 ```shell
 systemctl start apisix
 systemctl stop apisix
 ```
 
-如果通过其他方法安装，可以参考 [配置文件模板](https://github.com/api7/apisix-build-tools/blob/master/usr/lib/systemd/system/apisix.service) 进行修改，并将其放置在 `/usr/lib/systemd/system/apisix.service` 路径下。
+如果你是通过其他方法安装的 APISIX，可以参考[配置文件模板](https://github.com/api7/apisix-build-tools/blob/master/usr/lib/systemd/system/apisix.service)进行修改，并将其添加在 `/usr/lib/systemd/system/apisix.service` 路径下
```
Comment (SylviaBABY) [inline docs/zh/latest/how-to-build.md:243]:
```suggestion
如果你是通过其他方法安装的 APISIX，可以参考[配置文件模板](https://github.com/api7/apisix-build-tools/blob/master/usr/lib/systemd/system/apisix.service)进行修改，并将其添加在 `/usr/lib/systemd/system/apisix.service` 路径下。
```

File: docs/zh/latest/how-to-build.md [hunk 7]
```diff
@@ -21,345 +26,220 @@ title: 如何构建 Apache APISIX
...
+
+请参考[入门指南](./getting-started.md)了解如何使用 APISIX。
```
Comment (SylviaBABY) [inline docs/zh/latest/how-to-build.md:245]:
```suggestion
如需了解后续 APISIX 的使用，请参考[入门指南](./getting-started.md)获取更多信息。
```

**Review by moonming — CHANGES_REQUESTED** [review]
I think the previous directory structure is correct,  according to different installation methods such as docker, RPM, and source code.

**Review by moonming — COMMENTED** [review]
the file name is "how to build", which is means build from source code, not how to install.
And I think we can add doc for how to install.

**Comment by hf400159** [comment]
> the file name is "how to build", which is means build from source code, not how to install. And I think we can add doc for how to install.

@moonming 
1. The file name of `how-to-build` has not been changed because it is currently referenced in some documents, so it has not been changed for the time being. I will investigate and solve this problem later.
2. For the directory structure, it will be better to use the advanced features of Docusaurus to present the way, which allows the user to see the installation method he wants more intuitively.
3. For the way to build from source. We can probably move it to the GitHub repo as users installing from source would likely be checking on GitHub.
If you have other suggestions, welcome to discuss.

**Comment by juzhiyuan** [comment]
> > the file name is "how to build", which is means build from source code, not how to install. And I think we can add doc for how to install.
> 
> @moonming
> 
> 1. The file name of `how-to-build` has not been changed because it is currently referenced in some documents, so it has not been changed for the time being. I will investigate and solve this problem later.
> 2. For the directory structure, it will be better to use the advanced features of Docusaurus to present the way, which allows the user to see the installation method he wants more intuitively.
> 3. For the way to build from source. We can probably move it to the GitHub repo as users installing from source would likely be checking on GitHub.
>    If you have other suggestions, welcome to discuss.

@moonming
 
1. The inconsistency between the **filename** and the** sidebar label** was a long time ago. This PR is only responsible for the internationalization and synchronization of English changes from https://github.com/apache/apisix/pull/6964;
2. The problem of inconsistency between the file name and the title should be corrected **in the new PR**. As for the historical link reference problem, @SkyeYoung could help to redirect the client to the document site to solve it. The file name can be modified to `installation.md`;
3. Building from source can be explained in a separate document (e.g., README, separate documentation for installation from source, adding **a new Tab in current page** to provide "How to build from source code").

cc @hf400159 @navendu-pottekkat @kwanhur @spacewander @SylviaBABY @yzeng25 @membphis

**Comment by moonming** [comment]
> * For the directory structure, it will be better to use the advanced features of Docusaurus to present the way, which allows the user to see the installation method he wants more intuitively.

You didn't get my point. I mean, the user installs either via docker or via rpm, only one way. Therefore, the first-level directory must be the installation method.

**Comment by moonming** [comment]
> * The inconsistency between the **filename** and the **sidebar label** was a long time ago. This PR is only responsible for the internationalization and synchronization of English changes from [docs: update "Installation" guide #6964](https://github.com/apache/apisix/pull/6964);

please create a issue for this, thx

**Comment by moonming** [comment]
> 1. This PR is only responsible for the internationalization and synchronization of English changes from [docs: update "Installation" guide #6964](https://github.com/apache/apisix/pull/6964);

Got it. @hf400159 please add this desc in this PR, Otherwise others do not know

**Comment by moonming** [comment]
> > > the file name is "how to build", which is means build from source code, not how to install. And I think we can add doc for how to install.
> > 
> > 
> > @moonming
> > 
> > 1. The file name of `how-to-build` has not been changed because it is currently referenced in some documents, so it has not been changed for the time being. I will investigate and solve this problem later.
> > 2. For the directory structure, it will be better to use the advanced features of Docusaurus to present the way, which allows the user to see the installation method he wants more intuitively.
> > 3. For the way to build from source. We can probably move it to the GitHub repo as users installing from source would likely be checking on GitHub.
> >    If you have other suggestions, welcome to discuss.
> 
> @moonming
> 
> 1. The inconsistency between the **filename** and the **sidebar label** was a long time ago. This PR is only responsible for the internationalization and synchronization of English changes from [docs: update "Installation" guide #6964](https://github.com/apache/apisix/pull/6964);
> 2. The problem of inconsistency between the file name and the title should be corrected **in the new PR**. As for the historical link reference problem, @SkyeYoung could help to redirect the client to the document site to solve it. The file name can be modified to `installation.md`;
> 3. Building from source can be explained in a separate document (e.g., README, separate documentation for installation from source, adding **a new Tab in current page** to provide "How to build from source code").
> 
> image
> 
> cc @hf400159 @navendu-pottekkat @kwanhur @spacewander @SylviaBABY @yzeng25 @membphis

image
I mean we don't need to install etcd if install from docker.

**Comment by hf400159** [comment]
@juzhiyuan Since the documentation for building from source has not yet been created, there is no building from source tag added to this documentation. We will update this document later, not now. 

@moonming I will modify the content of the document so that the first level directory is the installation method.

**Review by SylviaBABY — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Comment by moonming** [comment]
is there a preview link for docs？

**Comment by SkyeYoung** [comment]
@hf400159 Hi, we can use https://surge.sh/ to publish a preview quickly. 😊

**Comment by hf400159** [comment]
preview
@moonming I have no way to upload to suger because of my network issues. You can take a look at this screenshot.

**Comment by moonming** [comment]
> @hf400159 Hi, we can use https://surge.sh/ to publish a preview quickly. 😊 (follow [README Guide](https://github.com/apache/apisix-website#basic), build, then use surge publish)

please create issue for this

**Review by moonming — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Comment by hf400159** [comment]
> > @hf400159 Hi, we can use https://surge.sh/ to publish a preview quickly. 😊 (follow [README Guide](https://github.com/apache/apisix-website#basic), build, then use surge publish)
> 
> please create issue for this

cc @moonming    issue link [1076]https://github.com/apache/apisix-website/issues/1076

**Pull request merged**